cmd/dynamic-networks-controller/networks-controller.go

```diff
@@ -146,5 +148,9 @@ func handleSignals(stopChannel chan struct{}, signals ...os.Signal) {
 
 func newContainerRuntime(configuration *config.Multus) (cri.ContainerRuntime, error) {
 	const withoutTimeout = 0
+	if configuration.CriType == cri.Crio {
+		crioTimeout := 5 * time.Second
```

Comment:
Why don't we need this for containerd ? maybe it's better to pass this as a "cmd" argument, so users decide what timeout they want.

Reply:
I have no clue why. I can only say a 0 will not work there - you get a `Context deadline exceeded` error instantly. 

Going forward I agree this could be an interesting parameter to have in the configuration. Right now, I think it doesn't make much sense, given the early stage of the project.

I'll unify these for now.

Reply:
Actually, I do know why (just had to look).

If you pass a 0, containerd [defaults to 10 seconds](https://github.com/maiqueb/multus-dynamic-networks-controller/blob/c7bd88d5d0a36205ae90cf1e91027605876ae538/vendor/github.com/containerd/containerd/client.go#L91). Why 10 ? No clue.